viam-cartographer.go

```diff
@@ -54,6 +55,7 @@ const (
 	parsePortMaxTimeoutSec               = 60
 	localhost0                           = "localhost:0"
 	defaultCartoFacadeTimeout            = 5 * time.Second
+	chunkSizeBytes                       = 1 * 1024 * 1024
```

Comment:
this should be const

Reply:
isnt it a const if it is in the const block?

Reply:
🤦 yes, yes it is... ignore!